src/components/PeopleFilters.tsx

```diff
@@ -1,18 +1,122 @@
+import { useSearchParams } from 'react-router-dom';
+import { Sex } from '../types';
+import cn from 'classnames';
+
 export const PeopleFilters = () => {
+  const [searchParams, setSearchParams] = useSearchParams();
+
+  const query = searchParams.get('query') || '';
+
+  const handleQuery = (event: React.ChangeEvent<HTMLInputElement>) => {
+    const params = new URLSearchParams(searchParams);
+
+    params.set('query', event.target.value);
+    setSearchParams(params);
+  };
```

Comment:
you need to make sure, that if query is empty, it should be deleted from searchParams